src/Azure.Functions.Cli/Helpers/DotnetHelpers.cs

```diff
@@ -99,6 +99,9 @@ public static async Task DeployDotnetFunction(string templateName, string functi
             "servicebusqueuetrigger" => "squeue",
             "servicebustopictrigger" => "stopic",
             "timertrigger" => "timer",
+            "daprpublishoutputbinding" => "daprPublishOutputBinding",
+            "daprserviceinvocationtrigger" => "daprServiceInvocationTrigger",
```

Comment:
The casing seems to be off. The key should match with the values that GetTemplates() method is returning.

Reply:
Changing the value to lower case, resulting in error. Template names are camel casing in the templates, so changing it here, it will result in error:

![image](https://github.com/Azure/azure-functions-core-tools/assets/16832724/11c920d0-fa38-4db3-89ea-ef973e0c3f92)